docs/developer/services/api_gateway.rst

```diff
@@ -7,5 +7,51 @@ API gateway
     :depth: 1
     :local:
 
-The API gateway serves as a middle layer between the Renku clients and the
-different backend services.
+The API gateway acts as a middle layer between the Renku clients and the
+different backend services (GitLab, Jupyterhub, Renku storage service, ...),
+thereby providing a single uniform interface to all backend services. More specifically,
```

Comment:
maybe  the rest of the sentence starting at "thereby" could be dropped? Would be simpler and avoid repetition

Reply:
removed.